x/tokenomics/token_logic_module/validator_distribution_test.go

```diff
@@ -0,0 +1,263 @@
+package token_logic_module
+
+import (
+	"context"
+	"testing"
+
+	"cosmossdk.io/log"
+	"cosmossdk.io/math"
+	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/mock/gomock"
+
+	"github.com/pokt-network/poktroll/app/pocket"
+	"github.com/pokt-network/poktroll/testutil/sample"
+	"github.com/pokt-network/poktroll/testutil/tokenomics/mocks"
+	tokenomicstypes "github.com/pokt-network/poktroll/x/tokenomics/types"
+)
+
+func TestDistributeValidatorRewards_Success(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	// Create mock staking keeper
+	mockStakingKeeper := mocks.NewMockStakingKeeper(ctrl)
+
+	// Create test validators with different stakes
+	validator1 := stakingtypes.Validator{
+		OperatorAddress: sample.ValOperatorAddressBech32(),
+		Tokens:          math.NewInt(700000), // 70% of total stake
```

Comment:
Can you change this to something like:

```go
var totalStake := 1_000_000
var val1Proportion = 0.7
```

Then, at the bottom, when you do `require.InDelta`, we can reuse the same ratios.